components/org.wso2.carbon.identity.organization.management.role.management.service/src/main/java/org/wso2/carbon/identity/organization/management/role/management/service/constant/RoleManagementConstants.java

```diff
@@ -42,6 +42,7 @@ public class RoleManagementConstants {
     public static final String UNION_SEPARATOR = " UNION ALL ";
 
     public static final String ORG_CREATOR_ROLE = "org-creator";
+    public static final String ORG_ADMINISTRATOR_ROLE = "Administrator";
```

Comment:
```suggestion
    public static final String ORG_ADMINISTRATOR_ROLE = "administrator";
```
As the org-creator is in simple case. @AnuradhaSK wdyt?

Reply:
Let's go with Administrator, I think we better rename org-creator to Organisation Creator in a separate effort